hack/build.sh

```diff
@@ -66,7 +66,7 @@ build(){
                 -X main.CtlName='${CTLNAME}' \
                 -X github.com/openebs/node-disk-manager/pkg/version.Version=${VERSION}" \
                 -o $output_name \
-               ./cmd
+               ./cmd/ndm_daemonset
```

Comment:
@satbirchhikara -- can this be taken as an argument -- just like the CTLNAME so that we don't need to create one more build script for operator.

Reply:
Done. Please have a look and let me know.